sympy/combinatorics/permutations.py

```diff
@@ -443,7 +444,7 @@ def __init__(self, *args):
                 for k, v in args[0].items():
                     self[k] = v
                 return
-        args = [int(a) for a in args]
+        args = [int(a) for a in args if a >= 0]
```

Comment:
```
args = [as_int(a) for a in args]  # import from compatibility -- it will raise a TypeError for non-int values
if any(a < 0 for i in args):
    raise ValueError
```